### PR TITLE
Feature/oui datagrid db cron and task

### PR DESCRIPTION
**Pull request by JDownloader**

### CODE REVIEW

File: client/app/hosting/module/hosting-module.controller.js [hunk 1]
```diff
@@ -35,38 +32,34 @@ angular.module("App").controller(
...
-                .finally(() => {
-                    if (_.isEmpty(this.modules.ids)) {
-                        this.loading = false;
-                    }
                 });
         }
 
-        transformItem (id) {
-            return this.HostingModule.getModule(this.$stateParams.productId, id)
+        transformItem (item) {
```
Comment (whispyy) [inline client/app/hosting/module/hosting-module.controller.js]: (Very Minor) : it is possible to pass object : transformItem ({ id }) and then call directly id

File: client/app/private-database/task/private-database-task.controller.js [hunk 2]
```diff
@@ -1,44 +1,32 @@
...
                 });
         }
 
-        getTasksDetails (id) {
-            return this.privateDatabaseService.getTaskDetails(this.productId, id);
-        }
-
-        onTransformItemsDone () {
-            this.loaders.tasks = false;
+        transformItem (item) {
```
Comment (whispyy) [inline client/app/private-database/task/private-database-task.controller.js]: (very minor) same transformItem ({ id })

File: client/app/hosting/module/MODULE.html [hunk 3]
```diff
@@ -1,9 +1,5 @@
 <div class="container-fluid px-0" data-ng-controller="HostingTabModulesController as ctrlHostingTabModules">
 
-    <div class="text-center" data-ng-if="ctrlHostingTabModules.loading">
-        <oui-spinner data-size="l"></oui-spinner>
-    </div>
-
     <div class="row" data-ng-show="!ctrlHostingTabModules.loading">
```
Comment (AxelPeter) [inline client/app/hosting/module/MODULE.html]: If the spinner has been removed, I think there is no need for the `ng-show` attribute :)

File: package.json [hunk 4]
```diff
@@ -76,7 +76,7 @@
     "less-plugin-remcalc": "0.0.1",
     "ovh-protractor-jasmine2-logs-reporter": "0.0.1",
     "ovh-ui-kit": "^2.0.0",
-    "ovh-ui-kit-bs": "git+https://github.com/ovh-ux/ovh-ui-kit-bs#v1.0.1",
+    "ovh-ui-kit-bs": "git+https://github.com/ovh-ux/ovh-ui-kit-bs#v1.0.3",
```
Comment (AxelPeter) [inline package.json]: It has been registered on NPM recently, we can just add ^1.0.3 now :)

**Pull request merged**